src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div data-qa="card" class="product-card">
+      
+      <img class="product-card__image" src="./images/imac.jpeg" alt="a1419iMac">
+
+      <div class="product-card__description">
+        Mоноблок APPLE A1419 iMac
+        27" Retina 5K (MNED2UA/A)
+      </div>
```

Comment:
```suggestion
      <div class="product-card__description">
        Mоноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </div>
```
use semantic tags - it is heading